src/ui.rs

```diff
@@ -528,46 +530,25 @@ impl<'a> UiFrame<'a> {
-                     strategies for mesh models, allowing designers to smoothly interpolate \
-                     between multiple mesh geometries and select the result with the most \
-                     desired features."));
-                ui.new_line();
-                regular_font_token.pop(ui);
-
-                ui.text(imgui::im_str!("DESCRIPTION"));
-                regular_font_token = ui.push_font(self.font_ids.regular);
-                ui.text_wrapped(
-                    imgui::im_str!(
-                    "The H.U.R.B.A.N. Selector software is a part of the SDC's Inolab \
+                    "The H.U.R.B.A.N. Selector is a part of the SDC's Inolab \
                     Department's plan to create a research platform for designers to \
                     test and verify new algorithms and create new forms.\n\
                     \n\
-                    H.U.R.B.A.N. Selector is a software which allows hybridization of \
-                    multiple 3D models with the target to find new aesthetic forms. \
-                    It is intended for designers as a tool for experimenting. Therefore, \
-                    it acts as a gateway to full-fledged parametric design software. \
+                    H.U.R.B.A.N. Selector is an experimental software for \
+                    hybridization of multiple 3D models with an aim to find new \
+                    aesthetic forms. \
+                    It serves as a gateway to full-fledged parametric design software. \
                     A user builds an operation pipeline where each stacked operation allows \
                     reconfiguration anytime influencing inputs and outputs of subsequent \
-                    operations.\n\
-                    \n\
-                    The program extends the creative possibilities of designers and helps \
+                    operations. The program extends the creative possibilities of designers and helps \
                     them create beyond the limits of their imagination given by memory/brain \
                     capacity as well as the ability to create different variations of form and \
-                    compositions. Using H.U.R.B.A.N. Selector provides designers with more \
-                    space for selection and algorithmic seeking of new forms. \n\
-                    \n\
-                    H.U.R.B.A.N. Selector is currently under development. It utilizes the Rust \
-                    programming language, and its source code is freely available to the community."));
+                    compositions.\n\
+                     \n\
+                     The software is currently in very early development stages. It strives to be \
+                     a tool for simple parametric modeling using various \
+                     hybridization strategies for mesh and voxel models, allowing designers to \
+                     smoothly interpolate between multiple source models."));
```

Comment:
The description has changed. This is what we exhibit.

Reply:
Should we also update the initial README section to this text?

Reply:
Not yet. We'll come back to it before the end of the project. 
We'll have to mention the sponsors too... Let's postpone.